tests/unit/components/Editor/ItemEditor.spec.js

```diff
@@ -417,6 +440,18 @@ describe("ItemEditor.vue", () => {
       ).toBeTruthy();
     });
 
+    it("check for survey mode", async () => {
+      await wrapper
+        .find('[data-test="surveyQuestionCheckbox"]')
+        .setChecked("checked");
+      await flushPromises();
+      expect(
+        wrapper.find('[data-test="subjectiveQuestionContainer"]').exists()
+      ).toBeTruthy();
+      expect(wrapper.find('[data-test="surveyOptions"]').exists()).toBeFalsy();
```

Comment:
options were anyways not being shown for subjective questions right?

Reply:
I told her to work on the tests later on when all things are finished, so I guess ignore this for now
It'll be like a separate small sprint for her - 2-3 days of writing tests

Reply:
okay